src/mongo/db/storage/csv_file_test.cpp

```diff
@@ -0,0 +1,479 @@
+#include <unistd.h>
+#include <vector>
+
+#include "csv_file.h"
+#include "mongo/bson/json.h"
+#include "mongo/unittest/assert.h"
+
+namespace mongo {
+using namespace fmt::literals;
+
+class CsvFileInputTest : public unittest::Test {
+protected:
+    void setUp() override {
+        system("cp -rn src/mongo/db/storage/csv_test /tmp/");
+    }
+};
+
+TEST_F(CsvFileInputTest, CsvBasicRead) {
+    CsvFileInput input("csv_test/basicRead.csv", "csv_test/basicRead.txt");
+
+    std::vector<mongo::BSONObj> expected = {
+
```

Comment:
You can do this:
```suggestion
    std::vector expected{
```
The compiler will be able to deduce the element type of the `vector`.

Reply:
Done